Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -45,10 +45,11 @@ def self.typecheck_args
   def self.typecheck
```

Comment:
I think it'd be good that changes to this file always trigger the `sorbet` workflow.